src/commands/database/utils.ts

```diff
@@ -0,0 +1,261 @@
+import { createRequire } from 'module'
```

Comment:
Note for the future: We have a schema for Jigsaw, we can auto-generate a client for this. We have an example of how to do so in the SDK